src/components/OrgSettings/AgendaItemCategories/Preview/AgendaCategoryPreviewModal.test.tsx

```
@@ -0,0 +1,122 @@
+    render(
+      <AgendaCategoryPreviewModal
+        agendaCategoryPreviewModalIsOpen={true}
+        hidePreviewModal={mockOnClose}
+        showUpdateModal={mockShowUpdateModal}
+        toggleDeleteModal={mockToggleDeleteModal}
+        formState={mockCategory}
+        t={mockT}
+      />,
+    );
+
+    const closeButton = screen.getByTestId('previewAgendaCategoryModalCloseBtn');
```

ESLint

[error] Replace `'previewAgendaCategoryModalCloseBtn'` with `⏎······'previewAgendaCategoryModalCloseBtn',⏎····`

(prettier/prettier)